src/DataViews/DataViewTestUtils.cpp

```diff
@@ -55,6 +57,15 @@ void CheckCopySelectionIsInvoked(const FlattenContextMenu& context_menu,
   EXPECT_EQ(clipboard, expected_clipboard);
 }
 
+void ExpectSameLines(const std::string& actual, const std::string& expected) {
+  static const std::string kDelimeter = "\r\n";
+
+  std::vector<std::string> actual_lines = absl::StrSplit(actual, kDelimeter);
+  std::vector<std::string> expected_lines = absl::StrSplit(expected, kDelimeter);
+  std::sort(std::begin(actual_lines), std::end(actual_lines));
```

Comment:
Is that sorting really needed? I was under the impression that `testing::UnorderedElementsAreArray` is taking care of that.